pycognaize/login.py

```diff
@@ -0,0 +1,78 @@
+import json
+import logging
+import os
+from glob import glob
+
+import requests
+import tempfile
+
+from pycognaize.common.exceptions import AWS_connection_exception
+
+
+class login:
+    """Allows to access the AWS S3 bucket using cognaize credentials"""
+
+    def __init__(self, username, password):
+        self._login(username, password)
+
+    def _login(self, email: str = '', password: str = ''):
+        """Get AWS access credentials and store in file"""
+        # url = CFG.host + CFG.API_ENDPOINT
+        HOST = os.environ.get('HOST')
+        url = f"{HOST}/api/v1/integration/storage/token"
+
+        authentication = {'email': email,
+                          'password': password}
+
+        try:
+            user_credentials_response = requests.post(url, json=authentication)
+        except requests.exceptions.ConnectionError:
+            logging.info(f'Failed connecting to url: {url}')
+            raise AWS_connection_exception
+        except requests.exceptions.Timeout:
+            logging.info(f'Connection timed out: {url}')
+            raise AWS_connection_exception
+
+        user_credentials = user_credentials_response.json()
+        if user_credentials_response.status_code == 200:
+            credentials_file = self._create_aws_config_file()
+            self._write_aws_config_file(user_credentials, credentials_file)
+        elif user_credentials_response.status_code == 403:
+            logging.info('You are not allowed to download data')
+            raise AWS_connection_exception("data download permission error")
```

Comment:
Remove logging when error is raised. More detailed error messages, with suggested solution.